File: qlty-cli/tests/lang/ruby/basic.in/lines.rb
def f1
    source_file = Tempfile.new(["ruby", ".kt"])
    source_file.write("foo(*args)")
    tree = source_file.parse

    bar
end

# Foo
def f2
    bar # does not count as comment line
end

# multi-line comment
=begin

line1
line2

line4
=end

def f3
    bar
end

File: qlty-cli/tests/lang/typescript/basic.in/lines.ts
function f1(): any {
  const sourceFile = new Tempfile(["ruby", ".kt"]);
  sourceFile.write("foo(...args)");
  const tree = sourceFile.parse();

  bar();
}

// Foo
function f2(): any {
  bar(); // does not count as comment line
}

// multi-line comment
/*

line1
line2

line4
*/

function f3(): any {
  bar();
}
